dashboard/frontend/src/ComponentCard.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import { Button, Card, CardHeader, CardBody, TextContent,Text } from '@patternfly/react-core'
+import { RebootingIcon ,ErrorCircleOIcon, OkIcon } from '@patternfly/react-icons';
+
+class ComponentCard extends Component {
+  static defaultProps = {
+    name: "default name",
+    url: ""
```

Comment:
I see that down in the code, we use `endpoint`, not `url`. Should we change it?
```suggestion
    endpoint: ""
```